media/transcoder/api/Transcoder.Samples/CreateJobWithAnimatedOverlay.cs

```diff
@@ -98,14 +94,8 @@ public class CreateJobWithAnimatedOverlaySample
                     X = 0.5,
                     Y = 0.5
                 },
-                StartTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
-                {
-                    Seconds = 12
-                },
-                EndTimeOffset = new Google.Protobuf.WellKnownTypes.Duration
-                {
-                    Seconds = 15
-                }
+                StartTimeOffset = Duration.FromTimeSpan(new TimeSpan(0, 0, 0, 12, 0)),
```

Comment:
And in this case it would be `TimeSpan.FromSeconds(12)`.
(I do wonder whether we should add `Duration.FromSeconds`, `Duration.FromMilliseconds` etc in the standard protobuf library, actually...)